spec/javascripts/components/feedback-spec.js

```diff
@@ -735,6 +738,56 @@ describe('Feedback component', function () {
     })
   }
 
+  describe('testing honeypot metadata on the "report a problem" form', function () {
+    beforeEach(function () {
+      jasmine.clock().install()
+      loadFeedbackComponent()
+      $('.js-something-is-wrong')[0].click()
+    })
+
+    afterEach(function () {
+      jasmine.clock().uninstall()
+    })
+
+    it('has an incrementing timer field', function () {
+      var $form = $('.gem-c-feedback #something-is-wrong')
+      var $timer = $form.find('input[name=timer]')
+      expect($timer.val()).toBe('0')
+      jasmine.clock().tick(1000)
+      expect($timer.val()).toBe('1')
+      jasmine.clock().tick(3000)
+      expect($timer.val()).toBe('4')
+    })
+
+    it('has a paste detection field', function () {
+      var $form = document.querySelector('.gem-c-feedback #something-is-wrong')
+      var $pastes = $form.querySelector('input[name=pastes]')
+      var $whatDoing = $form.querySelector('[name=what_doing]')
+      var $whatWrong = $form.querySelector('[name=what_wrong]')
+
+      var pasteEvent = new window.ClipboardEvent('paste')
+
+      $whatDoing.dispatchEvent(pasteEvent)
+      $whatWrong.dispatchEvent(pasteEvent)
```

Comment:
Another non-blocking comment. We have a [script for creating events](https://github.com/alphagov/govuk_publishing_components/blob/main/app/assets/javascripts/govuk_publishing_components/lib/trigger-event.js) that you might find useful here, might be able to write as:

```
window.GOVUK.triggerEvent($whatDoing, 'paste')
window.GOVUK.triggerEvent($whatWrong, 'paste')
```